packages/cli/src/index.ts

```diff
@@ -53,6 +53,10 @@ program
   )
   .option('-d, --dry', 'dry run (no changes are made to files)')
   .option('--run-in-band', 'run serially in the current process')
+  .option(
+    '--registry <value>',
+    'Define a registry where the package should be fetched from',
```

Comment:
Could you please also add a reference to the API docs 🙏 
https://github.com/CodeshiftCommunity/CodeshiftCommunity/blob/main/website/docs/api/codeshift-cli.mdx